### DISCUSSION TITLE
Question Regarding Status Messages SC 4.1.3 and Value Updates

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#3544] [Q&A] [answered]
I am reaching out to discuss a scenario regarding the functionality tied to the + and - buttons that update item values and amounts.

Upon activation of these buttons,  the item and amount values are automatically updated. My question pertains to whether this updated value must be automatically announced for Status Messages SC 4.1.3
![image](https://github.com/w3c/wcag/assets/115781693/5ec6c1fb-ed0a-4651-a2dd-4a20cf0f66c4)

**Comment by JAWS-test** [comment]
Whether the sum of the costs must be output per live region - no idea. But the value of the current form field (i.e. the number of products) should be perceptible when using the screen reader. An spinbutton is best suited for this (`role=spinbutton` or `input type=number`). A realisation via + and - button is not optimal, because then I cannot automatically perceive the value

**Comment by patrickhlauke** [comment, marked as answer]
Short answer would be: yes, those are dynamic updates / changes of content that to a sighted user that is not also running screen magnification (which may cut it off) or similar are immediately obvious/perceivable. Those need to be announced dynamically under 4.1.3
